pkg/plugins/policies/meshhttproute/api/v1alpha1/meshhttproute.go

```diff
@@ -141,10 +141,13 @@ type RequestRedirect struct {
 	// +kubebuilder:validation:Enum=http;https
 	Scheme   *string          `json:"scheme,omitempty"`
 	Hostname *PreciseHostname `json:"hostname,omitempty"`
+	// Path defines parameters used to modify the path of the incoming request.
+	// The modified path is then used to construct the Location header.
```

Comment:
nitpick: does "Location" need to be capitalised? I know you want to signify that it's not any header just the "location" header. I think we try not to use case sensitive headers anywhere because they are lowercased by Envoy.

```suggestion
	// The modified path is then used to construct the location header.
```

Reply:
I copied this from Gateway API doc, but I don't mind having them lowercased for consistency